app/modules/intelligence/tools/code_query_tools/get_file_content_by_path.py

```
@@ -160,6 +162,16 @@ def _run(
                     with_line_numbers,
                     starting_line=start_line or 1,
                 )
+
+                # Truncate content if it exceeds character limits
+                original_length = len(content)
+                content = truncate_response(content)
+                if len(content) > 80000:
+                    logging.warning(
```

Ruff (0.14.10)

Undefined name `logging`

(F821)